src/server/apis/settings.ts

```diff
@@ -14,6 +14,13 @@ function getRouter(github: GitHub): express.Router {
         return;
     }
 
+    const scopes = request.cookies['scope'] ?
+      request.cookies['scope'].split(',') : [];
+    if ((scopes.indexOf('admin:org_hook') === -1 || scopes.indexOf('read:org') === -1)) {
+      response.status(400).send('Missing required scope.');
+      return;
+    }
```

Comment:
Can we instead store the scopes in the user model and perform the check there? Doesn't seem robust to check the cookie here.

Reply:
Done with a chunk of other tidy up - sorry.